server/v1/tests/generator.js

```diff
@@ -0,0 +1,101 @@
+
+const generator = {
```

Comment:
You can give this a more descriptive variable name like `mockData` and the file name `mockData.js`